idd/Energy+.idd.in

```diff
@@ -40370,12 +40371,17 @@ ZoneHVAC:TerminalUnit:VariableRefrigerantFlow,
        \autosizable
        \default autosize
        \note Supply air temperature from the supplemental heater will not exceed this value.
-  N12; \field Maximum Outdoor Dry-Bulb Temperature for Supplemental Heater Operation
+  N12, \field Maximum Outdoor Dry-Bulb Temperature for Supplemental Heater Operation
        \type real
        \maximum 21.0
        \default 21.0
        \units C
        \note Supplemental heater will not operate when outdoor temperature exceeds this value.
+  A19; \field Controlling Zone or Thermostat Location
```

Comment:
And finally air loop equipment need to know the controlling zone otherwise the TU will be set point controlled.